### ISSUE TITLE
Issue when run the app on device in android or ios 

### ISSUE THREAD

**Issue by vinod1988** [itinance/react-native-fs#1223]
react-native-fs = 2.20.0
React native = "react-native": "0.70.14",

Here is stack trace

> Task :react-native-fs:compileDebugJavaWithJavac FAILED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.6/userguide/command_line_interface.html#sec:command_line_warnings
295 actionable tasks: 156 executed, 139 up-to-date
Note: /Users/vinod/workspace/vino/sk/yokohama/source_code/Yokohama/node_modules/@react-native-async-storage/async-storage/android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStorageModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/vinod/workspace/vino/sk/yokohama/source_code/Yokohama/node_modules/@react-native-async-storage/async-storage/android/src/javaPackage/java/com/reactnativecommunity/asyncstorage/AsyncStoragePackage.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: [2] Wrote GeneratedAppGlideModule with: [com.bumptech.glide.integration.okhttp3.OkHttpLibraryGlideModule, com.dylanvann.fastimage.FastImageOkHttpProgressGlideModule]
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
/Users/vinod/workspace/vino/sk/yokohama/source_code/Yokohama/node_modules/react-native-fs/android/src/main/java/com/rnfs/RNFSManager.java:119: error: cannot find symbol
    return android.os.Build.VERSION.SDK_INT <= android.os.Build.VERSION_CODES.P ? "w" : "rwt";
                                                                             ^
  symbol:   variable P
  location: class VERSION_CODES
Note: /Users/vinod/workspace/vino/sk/yokohama/source_code/Yokohama/node_modules/react-native-fs/android/src/main/java/com/rnfs/RNFSManager.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-fs:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 58s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.

**Comment by birdofpreyru** [comment]
[#1197 &mdash; The future of **react-native-fs**](https://github.com/itinance/react-native-fs/issues/1197)

**TL;DR.:** This repo is stale, no active work happening here now. I actively maintain a fork of the project, but it lives, and it is managed through a different repo &mdash; https://github.com/birdofpreyru/react-native-fs.
